### ISSUE TITLE
Header is covered by Navbar when Navigated to via Anchor in Sidebar (Profile)

### ISSUE THREAD

**Issue by KateAThornley** [CDH-Studio/I-Talent#365]
**Describe the bug**
When a user clicks through the anchors on the sidebar of the profile page, the header is not shown/are covered by the nav bar when the page scrolls down to the clicked anchor section

**To Reproduce**
1. Go to any profile on I-Talent
2. Click on an anchor in the sidebar for example: Skills
3. See how the Skills header is covered by the nav bar

**Expected behavior**
The header of the anchor clicked in the sidebar should be the first thing shown on the page

**Screenshots**
<img width="1327" alt="Screen Shot 2020-06-08 at 5 06 05 PM" src="https://user-images.githubusercontent.com/43188335/84080569-5b543080-a9aa-11ea-9d19-f058eb823245.png">

**Additional context**
This needs to be fixed to make I-Talent more accessible for all users and to improve user navigation.
